dockerfiles/keycloak/kc_realm_user.sh

```diff
@@ -37,7 +37,8 @@ if [ "${CHE_SELF__SIGNED__CERT}" != "" ]; then
     echo "${CHE_SELF__SIGNED__CERT}" > /scripts/openshift.cer
     keytool -importcert -alias HOSTDOMAIN -keystore /scripts/openshift.jks -file /scripts/openshift.cer -storepass openshift -noprompt
     keytool -importkeystore -srckeystore $JAVA_HOME/jre/lib/security/cacerts -destkeystore /scripts/openshift.jks -srcstorepass changeit -deststorepass openshift
-    /opt/jboss/keycloak/bin/jboss-cli.sh --file=/scripts/cli/add_openshift_certificate.cli && rm -rf /opt/jboss/keycloak/standalone/configuration/standalone_xml_history
+    /opt/jboss/keycloak/bin/jboss-cli.sh --file=/scripts/cli/add_openshift_certificate.cli && rm -f /opt/jboss/keycloak/standalone/configuration/standalone_xml_history
```

Comment:
```suggestion
```

Reply:
I don't think we need this line anymore.

Reply:
Yes, but I am not sure, if I have an issue if I remove this line.

Reply:
Could you try pls on your side?

Reply:
Yes, it works for me without this line, but I am not sure if it is never used.

Reply:
@davidfestal 
WDYT

Reply:
I prefer to keep both and add a comment for the first line.